plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataConfig.java

```diff
@@ -107,4 +112,19 @@ public JdbcMetadataConfig setDomainCompactionThreshold(int domainCompactionThres
         this.domainCompactionThreshold = domainCompactionThreshold;
         return this;
     }
+
+    @Min(1)
+    @Max(MAX_ALLOWED_INSERT_BATCH_SIZE)
+    public int getInsertBatchSize()
+    {
+        return insertBatchSize;
+    }
+
+    @Config("insert.batch-size")
```

Comment:
is it applicable to CREATE TABLE AS as well?

Reply:
Yes. The PageSink gets used there too. `write.batch-size`?

Reply:
sounds good!

Reply:
@hashhar @findepi @wendigo PTAL https://github.com/trinodb/trino/pull/8609